linchpin/FilterUtils/FilterUtils.py

```diff
@@ -301,3 +301,13 @@ def convert_to_json(input_str):
         return json.loads(input_str)
     else:
         return input_str
+
+
+def path_relative_to(path, base_path):
+    """
+    If `path` is not an OS filesystem absolute or relative path then assume
+    it is relative to `base_path`.
+    """
+    if path.startswith(('/', './', '../', '~/')):
```

Comment:
Why change OS standards that known by all?

Reply:
What do you mean?  Also, I should mention that this filter was the last one in a separate file, so I just moved the code in here with the rest of the filters.

Reply:
given a path which starts ('/', './', '../', '~/') then join it with `bath_path` path - this logic doesn't make sense because:
`/` - is absolute path
`./` - is current folder (relative)
`../` - is parent folder (relative)
`~/` - is home folder (relative)
Why change that behavior

Reply:
I looked at the code from which `path_relative_to` is called from and it essentially is meant to make a non-absolute path a relative path".  So if there's a path that looks like this: `provision/roles/aws` it appends the base path to it so it becomes `/path/to/linchpin/linchpin/provision/roles/aws`.  It's only used in the docker provider.  If the path starts with any of those then its just returned as-is.

Reply:
If it is only used by docker provider, why is it in the general FilterUtils and why is it part of AWS change?